clients/gtest/gemm_gtest.cpp

```diff
@@ -44,6 +44,8 @@ Representative sampling is sufficient, endless brute-force sampling is not neces
 const
 vector<vector<int>> matrix_size_range = {
                                         {-1, -1, -1, -1, 1, 1},
+                                        { 3, 33,  3,  3, 3, 3},
```

Comment:
33 is also giving errors, and Tim asked for this to be added as he uses it for batched gemm used for trsm